api/actions.py

```diff
@@ -69,7 +69,7 @@ def get_table_name(schema, table, restrict_schemas=True):
     if schema.startswith("_") or schema == "public" or schema is None:
         raise PermissionDenied
     if restrict_schemas:
-        if schema not in ["model_draft", "sandbox", "test"]:
+        if schema not in PLAYGROUNDS and schema not in UNVERSIONED_SCHEMAS:
```

Comment:
```suggestion
        if schema not in PLAYGROUNDS + UNVERSIONED_SCHEMAS:
```

Reply:
I am not sure which version I prefer. The first version performs better for large lists. Yours is undoubtedly more readable  and - according to `timeit` - more efficient for small lists. Interesting. As I expect small sets of schemas, I will go with your version.